cogs/fun_channel_checks.py

```diff
@@ -43,7 +43,16 @@ async def on_message(self, message):
                         break
                     await message.delete()
                     return
-
+            if message.content.lower() == "the end":
+                messages = await channel.history(limit=200)
+                fetch_message = messages.find(lambda m: m.author == client.user)
+                if fetch_message is None:
+                    await channel.send(f"{message.author.mention}, The last story was too large... Okay we are starting a new story, Let me start,\n\n{article}")
+                    return
+                story = " ".join(map(lambda m: m.content if (fetch_message.created_at < m.created_at) else '', messages))
+                await channel.send(story)
+                await channel.send(f"Okay we are starting a new story, Let me start,\n\n{article}")
+                return
```

Comment:
Over here you can do `return await channel.send(. . . . . )`, it'll look neater 👀 😃

Reply:
ah :mhm: